src/System.Threading.Thread/tests/System.Threading.Thread.Tests.csproj

```diff
@@ -15,8 +15,10 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Release|AnyCPU'" />
   <ItemGroup>
+    <Compile Include="CompressedStackTests.cs" />
     <Compile Include="ExceptionTests.cs" />
     <Compile Include="ThreadExceptionEventArgsTests.cs" />
+    <Compile Include="ThreadTests.netstandard1.7.cs" />
```

Comment:
I'd rather keep it the way it is, it's not immediately clear that the type existed before. At some point when we drop support for the older version I expect we would merge the files anyway.